### DISCUSSION TITLE
javascript/prototype-and-proto

### DISCUSSION THREAD

**Discussion by giscus** [howdy-mj/blog-v2#59] [Announcements]
# javascript/prototype-and-proto

마지막 업데이트: 2021.08.31

> 자바스크립트는 명령형, 함수형, 프로토타입 기반 객체지향 프로그래밍을 지원하는 멀티 패러다임 프로그래밍 언어다.

클래스의 가장 큰 특징은 상속이다. 상속은 객체지향 프로그래밍의 핵심 개념으로, 어떤 객체의 프로퍼...

https://www.howdy-mj.me/javascript/prototype-and-proto

**Comment by jungsikjeong** [comment]
와 저 정말 뻥안치고 모던자바스크립트홈페이지에서 봐도 이해가 잘 안갔는데 
님 정리글 보고 이해가 갔습니다..
선행학습으로 이해안가도 머릿속에 그냥 우겨넣고, 그다음 이해 안가는부분들을 다시 짚어가는식으로 하고있어서 이제와서 이해가 가는 거일수도있겠지만요:) 
여태 정리하신 다른 글들도 찬찬히 읽어보겠습니다! 정리를 정말 잘하셨네요 감사합니다!!
